reference/env_vars.rst

```diff
@@ -461,6 +461,8 @@ the :ref:`tools.msvc_build_command() <tools_msvc_build_command>`
 and the :ref:`MSBuild() <msbuild>` build helper, will not call ``devenv`` command to upgrade the ``sln`` project, irrespective of
 the ``upgrade_project`` parameter value.
 
+.. _env_vars_conan_sysrequires_mode:
+
```

Comment:
The default was changed in the PR, we need to update these docs.

Reply:
The default will still be the same as before unless you define the `default_mode` parameter in the constructor of `SystemPackageTool` and you don't define the variable.

Reply:
No, now the default is `None`. We don't know if someone is reading this environment variable to perform any other task 🍔